advcpmv-0.8-8.32.patch

```diff
@@ -13,7 +14,57 @@ diff -aur coreutils-8.32/src/copy.c coreutils-8.32-patched/src/copy.c
 +  struct stat src_open_sb;
 +};
 +
-+/* Begin progress Mod*/
++char * escape_double_quotes ( char * _cUnescapedString )
++{
++  char * _cEscapedString;
++  _cEscapedString = (char *) calloc( sizeof(char) * strlen(_cUnescapedString) * 2, sizeof(char) );
```

Comment:
Malloc was fine.

Reply:
Technically I think so, too. But malloc reserves uninitialized space, which is reported as error by valgrind. Calloc initializes the space, so it fixes those "errors".